.github/workflows/lint.yml

```diff
@@ -0,0 +1,16 @@
+name: Lint
+
+on: pull_request
```

Comment:
why is this one on PR but the other two are on push just curious

Reply:
That's a good question and my bad I didn't notice it before. I'm not entirely sure, I can ask Ally or someone else coz we j took the code from them lol